lib/cppcheck.cpp

```diff
@@ -659,8 +659,23 @@ unsigned int CppCheck::check(const std::string &path, const std::string &content
     return checkFile(Path::simplifyPath(path), emptyString, &iss);
 }
 
+void CppCheck::fileNotFoundError(const std::string& filename)
+{
+    mErrorLogger.reportErr(ErrorMessage({ErrorMessage::FileLocation(filename, 0, 0)},
+                                        filename,
+                                        Severity::error,
+                                        "File not found",
+                                        "fileNotFound",
+                                        Certainty::normal));
+    mExitCode = 1;
```

Comment:
We should not arbitrary set the exitcode. `Severrity::error` will already do this based on `--error-exitcode`. This will cause the analysis to fail even if the user suppresses this without any indication why.